packages/wrangler/src/queues/cli/commands/consumer/index.ts

```diff
@@ -2,7 +2,7 @@ import { createNamespace } from "../../../../core/create-command";
 
 export const queuesConsumerNamespace = createNamespace({
 	metadata: {
-		description: "Configure Queue consumers",
+		description: "Configure queue consumers",
```

Comment:
how come we are un-capitalising Queues in this PR? All our docs refer to Cloudflare `Q`ueues